doc/Language/objects.pod6

```diff
@@ -526,8 +523,7 @@ class MyClass {
 }
 =end code
 
-If all you need to do is checking things after the actual construction,
-or modify attributes after the object construction, writing a submethod
+If after object construction, you need to check things or modify attributes, writing a submethod
```

Comment:
Should not start with “If…”. Would be great to start with “The TWEAK submethod…”. A sentence after it can say “If after object construction”.